### ISSUE TITLE
Allow automatically tagging time

### ISSUE THREAD

**Issue by danielzfranklin** [aardappel/procrastitracker#48]
I'd like a means to automatically tag items. For instance, I'd like to automatically say that time spent in the Netflix or my news reader app is entertainment, and time spent on my school's website is schoolwork. I could see doing this with Regex, but I'm not particularly attached to that implementation idea. However, some means of automatic tagging would reduce the amount of time I have to spend tagging items, and thus help me be more productive.

**Comment by aardappel** [comment]
I agree that would be nice, and regexp would be a decent way to do it.

Not sure about the examples you give, given that all uses of netflix or your reader app are under one root node I presume, once you tag that node, any new entries will automatically be counted under that tag, e.g.

- watch: chrome.exe -> Netflix -> Some Movie
- tag Netflix as entertainment
- watch: chrome.exe Netflix -> Other Movie
- (Other Movie now tagged as entertainment).

in other words, tagging is hierarchical.
